### PR TITLE
Tone mapping setup guidance for 10.7

### PR DESCRIPTION
**Pull request by nyanmisaka**
- Tone mapping setup guidance for 10.7

### CODE REVIEW

File: general/administration/hardware-acceleration.md [hunk 1]
```diff
@@ -292,6 +292,40 @@ Stream #0:0 -> #0:0 (h264 (h264_cuvid) -> h264 (h264_nvenc))
 Stream #0:2 -> #0:1 (ac3 (native) -> aac (native))
 ```
 
+### Configuring Tone mapping
+
+Tone mapping with Nvidia NVENC and AMD AMF is through OpenCL image support.
+
+    > [!NOTE]
+    > Make sure the hardware acceleration is well configured before reading this section.
```
Comment (dkanada) [inline general/administration/hardware-acceleration.md:300]:
```suggestion
> [!NOTE]
> Make sure the hardware acceleration is well configured before reading this section.
```

File: general/administration/hardware-acceleration.md [hunk 2]
```diff
@@ -328,6 +328,7 @@ Docker      | OK           | untested | OK          | planned   | planned   | pl
    - For Intel iGPUs, follow the instructions from [intel-compute-runtime](https://github.com/intel/compute-runtime/releases). If you are using the docker image from jellyfin/jellyfin, this step can be skipped.
 
 > [!NOTE]
+> Tone mapping on Intel VAAPI needs an iGPU that support 10-bit decoding.
```
Comment (dkanada) [inline general/administration/hardware-acceleration.md:331]:
```suggestion
> Tone mapping on Intel VAAPI needs an iGPU that supports 10-bit decoding.
```

File: general/administration/hardware-acceleration.md [hunk 3]
```diff
@@ -292,12 +292,61 @@ Stream #0:0 -> #0:0 (h264 (h264_cuvid) -> h264 (h264_nvenc))
...
+OS/Platform | NVIDIA NVENC | AMD AMF  | Intel VAAPI | AMD VAAPI | Intel QSV | Software
+----------- | ------------ | -------- | ----------- | --------- | --------- | --------
+Linux       | OK           | OK       | OK          | planned   | planned   | planned
+Windows     | OK           | OK       | N/A         | N/A       | planned   | planned
+Docker      | OK           | untested | OK          | planned   | planned   | planned
+
+> [!NOTE]
+> Make sure the hardware acceleration is well configured before reading this section.
+
+1. On Windows: Install the latest Nvidia or AMD driver is fairly enough.
```
Comment (Camc314) [inline general/administration/hardware-acceleration.md:308]:
```suggestion
1. On Windows: Install the latest Nvidia or AMD drivers.
```

**Review by Camc314 — APPROVED** [review]

**Comment by nielsvanvelzen** [comment]
The linter was failing for this PR so it should not have been merged before that was fixed.

**Pull request merged**